mobile/native/src/db/mod.rs

```diff
@@ -310,7 +313,15 @@ pub fn get_payments() -> Result<Vec<(lightning::ln::PaymentHash, ln_dlc_node::Pa
 
     let payment_hashes = payments.iter().map(|(a, _)| a).collect::<Vec<_>>();
 
-    tracing::debug!(?payment_hashes, "Got all payments");
+    let formatted_payment_hashes = payment_hashes
+        .clone()
+        .iter()
+        .map(|hash| hex::encode(hash.0))
+        .collect::<Vec<_>>();
```

Comment:
🙃

given that on line 314 we only mapped the vector for the purpose of displaying, we don't need to do it twice.

We can remove the line above and use a combination of both:

```rust
let formatted_payment_hashes = payments.iter().map(|(hash, _)| hash).map(|hash| hex::encode(hash.0)).collect::<Vec<_>>();
```

Reply:
Good catch :)